proposals/2020-11-stargate-upgrade/README.md

```diff
@@ -0,0 +1,128 @@
+
+   - Provide a written report of the entire Stargate effort to enable Hub governance to make an informed decision.
+
+5. At conclusion, we will indicate why we have confidence that an upgrade won't be disruptive to the Hub’s ecosystem.
+
+## Stargate Plan Results
+
+------------
+
+### 1. Ecoystem Participation
+
+The results of ecosystem participation and engagement are available in our [Stargate Ecosystem Readiness Report](https://github.com/cosmosdevs/stargate/blob/master/ecosystem_readiness.md). We list on this page ecosystem partners such as Validators that provided their validator public keys as well as infrastructure partners that confirmed their integration testing the Stargate testnet tags. Validators, exchanges, and wallets have made up the most responsive of their results completing integration testing of the stargate releases.
+
+The Cosmos SDK, Tendermint, AiB, and IBC teams worked tirelessly to deliver the most robust software feature set that would make up the Cosmos Stargate release. Their review of their efforts are documented in their post ["How Seven Teams Collaborated To Deliver The Biggest Software Upgrade In The Cosmos Universe"](https://blog.cosmos.network/how-seven-teams-collaborated-to-deliver-the-biggest-software-upgrade-in-the-cosmos-universe-2288f4f9afe8).
+
+Integration testing continued through the release of the Cosmos SDK v0.40.0-rc2 that makes up the Cosmos-hub Stargate testnet tag for a simulated upgrade of the CosmosHub-3 to CosmosHub-4. Also, the AiB team continues to deliver simulation testing of Cosmos Stargate to ensure that any possible chain bug issues can be detected and documented.
+
+We are confident that we have and continue to achieve increased ecosystem engagement with the Stargate testnet.
+
+### 2. Critical Partner Support
+
+We delivered critical partner support to leading ATOM exchanges. Out of the exchange support efforts, we documented the [IBC Readiness Matrix](https://github.com/cosmosdevs/stargate/blob/master/ibc_readiness_matrix.md) that outlines the levels of readiness that exchanges may select as they upgrade to Cosmos Stargate.
+
+We also completed and delivered a completed legacy Amino Audit, [AminoRest & You](https://github.com/cosmosdevs/stargate/blob/master/audit.md). The core finding of this audit was that changes to the underlying structs result in an interface that is close to the prior version allowing legacy queries to return valid data.
+
+### 3. Documentation and Self-Certification
+
+Documentation was a success for the Cosmos Stargate effort. The upgrades with the most breaking changes such as legacy Amino have a complete audit with documentation on exceptions for blockchain API interfaces.
+
+Most exchanges and wallets have taken self-certification on directly. Our team continues to provide real-time support on multiple partner slack channels and on the Cosmos [#stargate Discord](https://discord.gg/W8trcGV) channel.
+
+### Cosmos Stargate Integration Success
```

Comment:
Do you want to more directly address results for (4) and (5) from the plan?

Reply:
Seems points (4) and (5) from the plan still have no counterpart under results. 

More specifically point (4) of the Plan says `   - Provide a written report of the entire Stargate effort to enable Hub governance to make an informed decision.` - what constitutes that report? 

And (5) says `At conclusion, we will indicate why we have confidence that an upgrade won't be disruptive to the Hub’s ecosystem.`. I can imagine someone implicitly concluding that if they have followed along but it might be good to also summarize this